src/ui/editor.js

```diff
@@ -1425,7 +1425,15 @@ class SurveyEditor extends Editor {
           });
 
         }
-      }
+      },
+      {
+        id    : 'export-to-csv',
+        text  : 'Export to CSV',
+        click : () => this.table.download('csv', this.survey.name + '.csv', { delimiter: ';' })
```

Comment:
please add the name of the cave to the CSV file's name in addition to the survey name